sdk/test-utils/recorder/src/baseRecorder.ts

```diff
@@ -128,13 +122,15 @@ export class NockRecorder extends BaseRecorder {
     super("node", testSuiteTitle, testTitle);
   }
 
-  public record(): void {
+  public record(recorderEnvironmentSetup: RecorderEnvironmentSetup): void {
+    this.environmentSetup = recorderEnvironmentSetup;
     nock.recorder.rec({
       dont_print: true
```

Comment:
can we change `dont_print: true` for `print: false`?

Reply:
Lol, this is an option provided by `nock` library, it doesn't provide an alternative name.

Reply:
ohhh :( too bad. Thank you!!!